RecoLocalCalo/Configuration/python/customizeEcalOnlyForProfiling.py

```diff
@@ -0,0 +1,51 @@
+import FWCore.ParameterSet.Config as cms
+
+# Customise the ECAL-only reconstruction to run on GPU
+#
+# Currently, this means running only the unpacker and multifit, up to the uncalbrated rechits
+def customizeEcalOnlyForProfilingGPUOnly(process):
+
+  process.consumer = cms.EDAnalyzer("GenericConsumer",
+      eventProducts = cms.untracked.vstring('ecalMultiFitUncalibRecHitGPU')
+  )
+
+  process.consume_step = cms.EndPath(process.consumer)
+
```

Comment:
seems like a lot of extra whitespace in this python file, remove where it is not needed.

Reply:
That's your opinion; mine differs.